src/core_plugins/kibana/public/management/sections/indices/_edit.html

```diff
@@ -83,19 +83,23 @@
       <!-- Search box -->
       <form role="form" class="kuiVerticalRhythm">
         <div class="kuiToolBar">
-          <input
-            aria-label="Filter"
-            ng-model="fieldFilter"
-            class="kuiLocalSearch fullWidth"
-            type="text"
-            placeholder="Filter"
-          />
-          <select data-test-subj="indexedFieldTypeFilterDropdown" class="kuiSelect" ng-if="state.tab == 'indexedFields'" ng-model="indexedFieldTypeFilter" ng-change="changeFilter('indexedFieldTypeFilter', indexedFieldTypeFilter)" ng-options="o for o in indexedFieldTypes">
-            <option value="">field type</option>
-          </select>
-          <select data-test-subj="scriptedFieldLanguageFilterDropdown" class="kuiSelect" ng-if="state.tab == 'scriptedFields'" ng-model="scriptedFieldLanguageFilter" ng-change="changeFilter('scriptedFieldLanguageFilter', scriptedFieldLanguageFilter)" ng-options="o for o in scriptedFieldLanguages">
-            <option value="">language</option>
-          </select>
+          <div class="kuiToolBarSearch">
+            <div class="kuiToolBarSearchBox">
+              <input
+                aria-label="Filter"
+                ng-model="fieldFilter"
+                class="kuiToolBarSearchBox__input"
+                type="text"
+                placeholder="Filter"
+              />
+            </div>
+            <select data-test-subj="indexedFieldTypeFilterDropdown" class="kuiSelect" ng-if="state.tab == 'indexedFields'" ng-model="indexedFieldTypeFilter" ng-change="changeFilter('indexedFieldTypeFilter', indexedFieldTypeFilter)" ng-options="o for o in indexedFieldTypes">
+              <option value="">field type</option>
+            </select>
+            <select data-test-subj="scriptedFieldLanguageFilterDropdown" class="kuiSelect" ng-if="state.tab == 'scriptedFields'" ng-model="scriptedFieldLanguageFilter" ng-change="changeFilter('scriptedFieldLanguageFilter', scriptedFieldLanguageFilter)" ng-options="o for o in scriptedFieldLanguages">
+              <option value="">language</option>
+            </select>
+          </div>
```

Comment:
I think we can make a few small adjustments to the UI to improve the information architecture:

1. If we move the search controls beneath the tabs, then they'll be closer to the content they're affecting.
2. If we switch the position of the `select` and the `input`, then the user will decide first what kind of content they want to filter on (strings, booleans, etc) and then they can decide what the search for. I think this is a little more of an intuitive flow.
3. I'm not sure ToolBar works well in this UI without upgrade the entire table to use the Table component, too. That's something we should do eventually, but that endeavor is significantly beyond the scope of this PR and I'd hate to block this feature. Instead, let's use some new components I'm introducing in https://github.com/elastic/kibana/pull/10713.

Here's how the UI will look with these suggestions implemented:

![image](https://cloud.githubusercontent.com/assets/1238659/23598020/6d28c1c2-01ec-11e7-85fe-1aef10da3d32.png)

![image](https://cloud.githubusercontent.com/assets/1238659/23598022/70414d2a-01ec-11e7-8279-57794ddebe69.png)

![image](https://cloud.githubusercontent.com/assets/1238659/23598030/7509d6ec-01ec-11e7-858f-1adb3da785a4.png)

Assuming we move forward with this, here's what our next steps will be:

1. Merge https://github.com/elastic/kibana/pull/10713.
2. Update this markup to be:
```html
<form role="form" class="kuiFieldGroup kuiVerticalRhythm">
  <div
    class="kuiFieldGroupSection"
    ng-if="state.tab == 'indexedFields'"
  >
    <label
      class="kuiLabel kuiFieldGroupSection__label"
    >
      Field type
    </label>

    <select
      data-test-subj="indexedFieldTypeFilterDropdown"
      class="kuiSelect kuiFieldGroupSection__control"
      ng-model="indexedFieldTypeFilter"
      ng-change="changeFilter('indexedFieldTypeFilter', indexedFieldTypeFilter)"
      ng-options="o for o in indexedFieldTypes"
    >
      <option value="">All field types</option>
    </select>
  </div>

  <div
    class="kuiFieldGroupSection"
    ng-if="state.tab == 'scriptedFields'"
  >
    <label
      class="kuiLabel kuiFieldGroupSection__label"
    >
      Language
    </label>

    <select
      data-test-subj="scriptedFieldLanguageFilterDropdown"
      class="kuiSelect kuiFieldGroupSection__control"
      ng-model="scriptedFieldLanguageFilter"
      ng-change="changeFilter('scriptedFieldLanguageFilter', scriptedFieldLanguageFilter)"
      ng-options="o for o in scriptedFieldLanguages"
    >
      <option value="">All languages</option>
    </select>
  </div>

  <div class="kuiFieldGroupSection kuiFieldGroupSection--wide">
    <label class="kuiLabel kuiFieldGroupSection__label">
      Search
    </label>

    <div class="kuiSearchInput kuiFieldGroupSection__control">
      <div class="kuiSearchInput__icon kuiIcon fa-search"></div>
      <input
        class="kuiSearchInput__input"
        type="text"
        aria-label="Filter"
        ng-model="fieldFilter"
        placeholder="Filter"
      >
    </div>
  </div>
</form>
```
3.  Move this markup in between the blocks tagged `Tabs` and `Tab content`.
4. Update the `Tab content` markup to be this (this just adds the VerticalRhythm component):
```html
<!-- Tab content -->
<indexed-fields
  ng-show="state.tab == 'indexedFields'"
  class="fields indexed-fields kuiVerticalRhythm"
></indexed-fields>

<scripted-fields
  ng-show="state.tab == 'scriptedFields'"
  class="fields scripted-fields kuiVerticalRhythm"
></scripted-fields>

<source-filters
  ng-show="state.tab == 'sourceFilters'"
  index-pattern="indexPattern"
  class="fields source-filters kuiVerticalRhythm"
></source-filters>
```
5. In `management/styles/main.less`, update the `fields` class on line 189 to have display: block:
```less
kbn-management-indices {
  .fields {
    display: block;
```